src/test/java/com/openlattice/conductor/ConductorBootstrap.java

```diff
@@ -31,29 +31,23 @@
  * @author Matthew Tamayo-Rios &lt;matthew@openlattice.com&gt;
  */
 public class ConductorBootstrap {
+
     protected static final Indexer INDEXER;
 
     static {
-        INDEXER = new Indexer();
         final var logger = LoggerFactory.getLogger( ConductorBootstrap.class );
-        if ( NetworkUtils.isRunningOnHost( "bamboo.openlattice.com" ) ) {
-            LoggerFactory.getLogger( ConductorBootstrap.class ).info( "Running on bamboo!" );
-            try {
+        INDEXER = new Indexer();
+        try {
+            if ( NetworkUtils.isRunningOnHost( "bamboo.openlattice.com" ) ) {
```

Comment:
not 100% relevant to this PR, but can we move this to configuration or dependency injection?

Reply:
I beliieeeve so though there are very few tests here and this class should probably be refactored/added into rhizome as a building block for testing startup for arbitrary services as it exists independently in Conductor, Linker and Indexer

Reply:
Yeah let's do that separately. Just jumped out at me as I was reading this.

Reply:
We have a configuration for testing called "awstest" that is set as an environment profile. Haven't tested it recently and is intended for providing CI-configurations on AWS. I suspect, it is broken since we moved Bamboo from outside to inside of GovCloud.

Reply:
https://jira.openlattice.com/browse/LATTICE-2349